client/packages/portal-core/src/portal-actions/portal-actions.ts

```diff
@@ -0,0 +1,26 @@
+import { BehaviorSubject } from 'rxjs';
+import { actions } from './portal-actions-config';
+import { PortalAction, PortalActions } from './types';
+
+const actions$ = new BehaviorSubject<PortalAction[]>(actions);
+
+const activeAction$ = new BehaviorSubject<PortalAction | undefined>(undefined);
+
+function setActiveActionById(actionId: string) {
```

Comment:
If you call this function with an actionId that doesnt exist, it will set activeAction to undefined. Is this the desired behaviour?

Reply:
maybe not ill fix ;)